### PR TITLE
English leak for "Close" in Autocomplete

### PR DESCRIPTION
**Pull request by stephanieyoshimoto-okta**
[OKTA-742653](https://oktainc.atlassian.net/browse/OKTA-742653)

## Summary

- Use Close key to fix "Close" English leak
- Add key for Open to be used to fix "Open" English leak

## Testing & Screenshots

- [ ] I have confirmed this change with my designer and the Odyssey Design Team.

With pseudo-loc language selected,
Before:
Screenshot 2024-07-12 at 11 25 47 AM

Now:
Screenshot 2024-07-12 at 11 23 24 AM

### CODE REVIEW

File: packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties [hunk 1]
```diff
@@ -2,6 +2,7 @@ breadcrumbs.home.text = Home
 breadcrumbs.label.text = Breadcrumbs
 close.text = close
```
Comment (mauriciocastillosilva-okta) [inline packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: Why is `close` all lower case? Shouldn't it be capitalized `Close`?

**Review by mauriciocastillosilva-okta — COMMENTED** [review]
One question

@KevinGhadyani-Okta or @bryancunningham-okta please take a look as well

File: packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties (hunk 1 above)
Comment (stephanieyoshimoto-okta) [reply packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: yeah I feel like it should be. is the change ok to leave in if I change the key in the properties file?

File: packages/odyssey-react-mui/src/Autocomplete.tsx [hunk 2]
```diff
@@ -484,6 +485,7 @@ const Autocomplete = <
       onChange={onChange}
       onInputChange={onInputChange}
       onFocus={onFocus}
+      // openText={t("open.text")} // OKTA-747967 Uncomment this line after translation is received
```
Comment (michaeltamaki-okta) [inline packages/odyssey-react-mui/src/Autocomplete.tsx:488]: Why do we have to wait for translations to be received? Is this a requirement for odyssey?

File: packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties (hunk 1 above)
Comment (michaeltamaki-okta) [reply packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: I'd argue it's ok. The letter casing issue isn't a breaking translation change

File: packages/odyssey-react-mui/src/Autocomplete.tsx (hunk 2 above)
Comment (mauriciocastillosilva-okta) [reply packages/odyssey-react-mui/src/Autocomplete.tsx:488]: Good point, we can actually merge the code, no need to comment it out

At the end, today we have English. If we merge this code, we will still have English, so no change there. Once we deliver translations, then all is fixed and we won't need to uncomment this at that time.

File: packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties (hunk 1 above)
Comment (mauriciocastillosilva-okta) [reply packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: Yes, it is not a breaking translation change as you said, but I think if it is used only for the tooltip, and the other tooltips are capitalized, we can be consistent. 
Stephanie, can you check if this key is used somewhere else?

Comment (stephanieyoshimoto-okta) [reply packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: it is also used in Dialog.tsx and Toast.tsx, but as aria labels. is it ok to change the key to upper case?

**Review by KevinGhadyani-Okta — APPROVED** [review]

File: packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties (hunk 1 above)
Comment (KevinGhadyani-Okta) [reply packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: I would use `"Close"`.

Comment (mauriciocastillosilva-okta) [reply packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties:3]: Yeah, let's change it to capitalized

**Pull request closed without merging**